Lesson_9/Tusk_1.py

```diff
@@ -0,0 +1,19 @@
+fib_sequence = [0, 1, 1]
+
+
+def fib_gen(sequence_end, list):
```

Comment:
You don't need to use some list outside, you can initialize needed values inside your generator. And it would be much better to use variables to keep current and previous values of sequence instead of list and override those variables on every iteration